frontend/src/about.js

```diff
@@ -9,29 +9,30 @@ export default () => (
     </h3>
     <p>
       This tool uses a machine learning model trained on 1.7 million{' '}
-      <a href='https://www.ncbi.nlm.nih.gov/pmc/'>PubMed Central</a> documents
-      to recommend suitable journals based on the textual content of your{' '}
-      <a href='https://www.biorxiv.org/'>bioRxiv</a> preprint. This tool
-      provides a "map" of all documents within{' '}
       <a href='https://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/'>
-        {' '}
-        PubMed Central's Open Access Corpus{' '}
-      </a>
-      , and plots where your <a href='https://www.biorxiv.org/'>bioRxiv</a>{' '}
-      preprint falls on this map. Each square bin on the map represents a group
-      of papers that share similar textual content. Click on a square to view
-      more information about papers contained in a given bin.
+        PubMed Central open access documents
+      </a>{' '}
+      to recommend suitable journals based on the textual content of your{' '}
+      <a href='https://www.biorxiv.org/'>bioRxiv</a> preprint.
+    </p>
+    <p>
+      The tool also provides a "map" of the documents, grouped into bins based
```

Comment:
```suggestion
      The tool also provides a "map" of PubMed Central Open Access documents, grouped into bins based
```

Reply:
Is "the PubMed Central documents" okay instead, since we already say "PubMed Central Open Access documents" in the previous sentence

Reply:
Trying to filter out as many words as possible to not info-overload the user

Reply:
Yeah that is fine.